impeller/compiler/shader_lib/impeller/gaussian.glsl

```diff
@@ -56,6 +61,10 @@ vec2 IPVec2FastGaussianIntegral(vec2 x, float sigma) {
   return 1.0 / (1.0 + exp(-kSqrtThree / sigma * x));
 }
 
+vec4 IPVec2FastGaussianIntegral(vec4 x, float sigma) {
```

Comment:
same here

Reply:
This one is integrating over the range x to y and w to z, docstring added.